Example/TABTestKit_ExampleUITests/ExampleFeature.swift

```diff
@@ -22,7 +22,42 @@ class ExampleFeature: TestBase {
       Then(I: seeTheExampleScreen)
     }
   }
+}
+
+class BiometricTests: XCTestCase {
+  
+  private var app: XCUIApplication!
+  
+  private lazy var authenticateButton = app.buttons.element(matching: .button, identifier: "authenticateButton")
```

Comment:
These should be properties on the ExampleScreen class I think, otherwise we're mixing two different styles of writing these tests which defeats the purpose of the screen object

Reply:
There's also no need for these to be `lazy vars`, they can just be `let`. The performance increase will never be noticeable because these are actually queries that only get executed when you actually use the element, like `.tap()` etc